install/README.md

```diff
@@ -363,6 +364,19 @@ and the expected response. Some digitisers have different nominal responses for
 | _Start_ | Gain start time|
 | _Stop_ | Gain stop time|
 
+#### _TELEMETRIES_ ####
+
+Sometimes the datalogger and the sensor are not at the same location. Usually this means there is some form of analogue link between the two, either a dedicated
+telephone line, or an FM radio link. This table allows this to be documented, and provides a mechanism to adjust the signal gains if known.
+
+| Field | Description | Units |
+| --- | --- | --- |
+| _Station_ | Datalogger recording _Station_|
+| _Location_ | Recording sensor site _Location_ |
```

Comment:
other tables (preamps, gains) also include subsource (and even sublocation...)
shall we add the column here to, even if not used?

Reply:
I was thinking of that too, swaying back and forwards related to what it does.  As this will not have a bias, should it be simply "Scale Factor" rather than bias.

(if a bias is added then the response becomes a polynomial and will make life tricky with responses).

Reply:
oh great!! As I was suggesting to change it to "scale factor" but then got scared that I was missing something obvious :D